scripts/assign.py

```
@@ -193,18 +194,19 @@ def main():
+                        all_issues_have_prs = False
+                        break
 
-                if issues_without_prs:
-                    # User has uncompleted issues
-                    issues_list = ", #".join(str(num) for num in issues_without_prs)
+                # If not all issues have PRs, block the assignment
+                if not all_issues_have_prs:
                     comment_body = (
-                        f"You can't take this task yet. You still have uncompleted issues: "
-                        f"#{issues_list}. Please complete them before requesting another."
+                        f"You can't take this task yet. You have uncompleted issues without open PRs. "
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)